vagrant/tutorial/cache.sh

```diff
@@ -17,3 +17,16 @@ for image in busybox clusterhq/mongodb dockerfile/redis clusterhq/flask; do
     docker pull "${image}"
 done
 
+
+## Hacks to demonstrate other changes
+systemctl unmask firewalld
+systemctl enable firewalld
+systemctl start firewalld
+for PORT in 3306 9200 5000 80 5432 27018; do # List of ports used in acceptance tests
+    for DEST in 172.16.255.240 172.16.255.241; do # List of vagrant nodes
+        # Extra rules for proxies
+        firewall-cmd --permanent --add-rule ipv4 filter FORWARD 0 --destination $DEST --protocol tcp --desitnation-port $PORT -j ACCEPT
+    done
+    # New rule for open ports
+    firewall-cmd --permanent --add-rule ipv4 filter INPUT 0 --protocol tcp --desitnation-port $PORT -j ACCEPT
+done
```

Comment:
I don't quite understand why these rules are being added.  I suppose they are necessary because we no longer simply disable the system firewall **and** the new iptables rules added by this design only accept traffic between the docker container and its host, not traffic between the two vagrant instances?

If that's so, then don't people installing Flocker on their own systems (eg CentOS 7) with a firewall still need to be told they have to allow certain traffic through the firewall (thus, there still needs to be something in the install documentation)?